lib/time/duration.ex

```diff
@@ -424,12 +545,25 @@ defmodule Timex.Duration do
   end
 
   @doc """
-  Time interval since Epoch.
+  Returns the amount of time since Epoch.
+
+  The argument is an atom indicating the type of time units to return.
+
+  The allowed unit type atoms are:
+  - :microseconds
+  - :milliseconds
+  - :seconds
+  - :minutes
+  - :hours
+  - :days
+  - :weeks
+
+  ## Examples
 
-  The argument is an atom indicating the type of time units to return (see
-  convert/2 for supported values).
+  iex> Timex.Duration.now(:seconds)
+  :os.system_time(:seconds)
```

Comment:
I think this test is prone to a race condition, which may be rare enough to not be a problem, but philosophically I don't like it. I feel like a better test is whether or not the value returned makes sense for the given date, e.g. `:os.system_time(:seconds) <= :calendar.datetime_to_gregorian_seconds({{1970,1,1},{0,0,0}}) + Timex.Duration.now(:seconds)`